content/news/2021-04-04-bevy-0.5/index.md

```diff
@@ -0,0 +1,1254 @@
+
+We have removed `ChangedRes<A>` in favor of the following:
+
+
+```rust
+fn system(a: Res<A>) {
+    if a.is_changed() {
+        // do something
+    }
+}
+```
+
+### Optional Resource Queries
+
+<div class="release-feature-authors">authors: @jamadazi</div>
+
+It is now possible for a system to check for Resource existence via `Option` queries:
+
+```rust
+fn system(a: Option<Res<A>>) {
+    if let Some(a) = a {
+        // do something
+    }
+}
+```
+
+### New Bundle Naming Convention
+
+Component Bundles previously used the `XComponents` naming convention (ex: `SpriteComponents`, `TextComponents`, etc). We decided to move to a `XBundle` naming convention (ex: `SpriteBundle`, `TextBundle`, etc) to be more explicit about what these types are and to help prevent new users from conflating Bundles and Components. 
+
+## Rich Text
+
+<div class="release-feature-authors">authors: @tigregalis</div>
+
+Text can now have "sections", each with their own style / formatting. This makes text much more flexible, while still respecting the text layout rules:
+
+![rich_text](rich_text.png)
+
+This is accomplished using the new "text section" api:
+
+```rust
+commands
+    .spawn_bundle(TextBundle {
+        text: Text {
+            sections: vec![
+                TextSection {
+                    value: "FPS: ".to_string(),
+                    style: TextStyle {
+                        font: asset_server.load("FiraSans-Bold.ttf"),
+                        font_size: 90.0,
+                        color: Color::WHITE,
+                    },
+                },
+                TextSection {
+                    value: "60.03".to_string(),
+                    style: TextStyle {
+                        font: asset_server.load("FiraMono-Medium.ttf"),
+                        font_size: 90.0,
+                        color: Color::GOLD,
+                    },
+                },
+            ],
+            ..Default::default()
+        },
+        ..Default::default()
+    })
+```
+
+## HIDPI Text
+
+<div class="release-feature-authors">authors: @blunted2night</div>
+
+Text is now rendered according to the current monitor's scale factor. This gives nice, crisp text at any resolution.
+
+![hidpi_text](hidpi_text.png)
+
+## Render Text in 2D World Space
+
+<div class="release-feature-authors">authors: @CleanCut, @blunted2night</div>
+
+Text can now be spawned into 2D scenes using the new `Text2dBundle`. This makes it easier to do things like "draw names above players".
+
+![2d_text](2d_text.gif)
+
+## World To Screen Coordinate Conversions
+
+<div class="release-feature-authors">authors: @aevyrie</div>
+
+It is now possible to convert world coordinates to a given camera's screen coordinates using the new `Camera::world_to_screen()` function. Here is an example of this feature being used to position a UI element on top of a moving 3d object. 
+
+<video controls loop><source src="world_to_screen.mp4" type="video/mp4"/></video>
```

Comment:
I made a small (63KB), smooth, looping video for this.

https://user-images.githubusercontent.com/2632925/113595672-2d517400-95ee-11eb-95d9-238aa0b624d2.mp4